env.example

```diff
@@ -0,0 +1,24 @@
+"""
+
+This file contains sample environment variables used across the project.
+
+Its main purpose is to track the new variables that have been added to the project.
+
+Incase you add any environment variable in the project endevour to add the sample
+in this file so that it enables other contributors to know which environment
+variables to set in there environments.
+
+NOTE:
+- Always set these variables in your environments before testing the project
+- Follow the convention where keys are UPPERCASE i.e DATABASE_HOST, DATABASE_NAME e.t.c
+- The values should just be samples where one will replace with their values 
+i.e your_db_name, your_db_username e.t.c
+
+"""
+# PostgreSQL connection variables
+
+DATABASE_NAME=your_db_name
```

Comment:
This is descriptive enough @reifred , but am wondering if you can add the real db name and the rest of the fields. Plus is there a way to make them secret , so that someone cant get them straight by accessing the env.example file?

Reply:
I get your point @NicholusMuwonge but as the description in the file. It just contains sample environment variables meaning I didn't reveal the real environment variables used. For the real environment variables, I created a **.env file** in my project where I include them. The file was used to enable one to know which variables I used for their project setup.